ArduSub/Sub.h

```diff
@@ -457,6 +457,8 @@ class Sub : public AP_HAL::HAL::Callbacks {
     void update_turn_counter();
     void read_AHRS(void);
     void update_altitude();
+    void init_beacon();
```

Comment:
I think that it is better to exclude the beacon related method definition with the preprocessor as well.